Sync `css/CSS2/visudet` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 2de0d0805816aa9e8e2697e209d330a456e5239b
<pre>
Sync `css/CSS2/visudet` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=301811">https://bugs.webkit.org/show_bug.cgi?id=301811</a>
<a href="https://rdar.apple.com/163866091">rdar://163866091</a>

Reviewed by Alan Baradlay.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f">https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f</a>

* LayoutTests/TestExpectations:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-005-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/crashtests/canvas-huge-min-max-sizes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-002.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-011-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-011.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-012-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-012.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-013-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-013.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-014-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-014.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-015-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-015.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-016-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-016.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-201-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-201.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-202-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-202.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-203-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-203.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-204-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-204.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-205-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-205.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-206-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-206.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/max-width-109.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-201-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-202-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-203-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-206-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/1x1-green.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-width-50.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/swatch-blue.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/width-50-no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/width-50-ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/w3c-import.log: Added.

Canonical link: <a href="https://commits.webkit.org/302446@main">https://commits.webkit.org/302446@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b0a5b718057d13502e440aaae321ac440b929c16

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/129113 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/1372 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/39949 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/136492 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/80491 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/2556da4a-b0da-4acb-8127-cfbc1b542a76) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/1304 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/1249 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/98309 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/66183 "") | [❌ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/be7a41ae-e305-4f96-87d5-b42af6b368ec) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/132060 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/162/builds/1012 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/115657 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/78954 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/382b4775-ecf5-4398-9983-d952d1afbc3f) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/935 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/33774 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/79771 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/109382 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/34270 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/138966 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/1165 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/1132 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/106845 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/1217 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/111993 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/106672 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/27157 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/957 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/30517 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/53724 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/1238 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/1064 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/1110 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/1162 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

alanbaradlay commented on LayoutTests/TestExpectations:
```diff
@@ -7867,3 +7867,12 @@ imported/w3c/web-platform-tests/page-lifecycle/child-out-of-viewport.tentative.h
 imported/w3c/web-platform-tests/page-lifecycle/set-composited-layer-position.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/cookies/partitioned-cookies/partitioned-cookies-a-b-a-embed.tentative.https.html [ Skip ]
+
+imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003.xht [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004.xht [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005.xht [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006.xht [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-20.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-40.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80.html [ ImageOnlyFailure ]
```
surprised we fail such basic/old tests.

Ahmad-S792 replied:
@alanbaradlay - these are failed by all browsers:


imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003.xht [ ImageOnlyFailure ]
imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004.xht [ ImageOnlyFailure ]
imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005.xht [ ImageOnlyFailure ]
imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006.xht [ ImageOnlyFailure ]

Rest, we should look into.

Ahmad-S792 committed:
Sync `css/CSS2/visudet` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=301811
rdar://163866091

Reviewed by Alan Baradlay.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/bc9587088f1e5621317f3db04b0f0d5f8e008b4f

* LayoutTests/TestExpectations:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-005-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/content-height-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/crashtests/canvas-huge-min-max-sizes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-applies-to-010a.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-computed-002.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-003a.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/height-percentage-004.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-001.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-002.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-003.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-004.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-005.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-006.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010-ref.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-010.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-011-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-011.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-012-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-012.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-013-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-013.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-014-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-014.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-015-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-015.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-016-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/inline-block-baseline-016.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-201-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-201.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-202-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-202.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-203-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-203.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-204-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-204.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-205-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-205.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-206-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/line-height-206.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/max-width-109.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/content-height-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-201-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-202-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-203-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/line-height-206-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-all-auto.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-max-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-20-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-20.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-height-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-min-width-80.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/replaced-elements-width-40.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/1x1-green.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/height-25-width-50.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/swatch-blue.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/width-50-no-ratio.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/support/width-50-ratio-2.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/CSS2/visudet/w3c-import.log: Added.

Canonical link: https://commits.webkit.org/302446@main
